feat: Add admin resume process endpoint

almerico opened the pull request:
fixes Activiti/Activiti#2151

almerico committed:
fixes #2151 adde Resume endpoint on task admin controller

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/signed)](https://cla-assistant.io/Activiti/activiti-cloud-runtime-bundle-service?pullRequest=145) <br/>All committers have signed the CLA.

almerico committed:
fix codacy warning

salaboy commented on activiti-cloud-services-runtime-bundle/activiti-cloud-services-rest-impl/src/test/java/org/activiti/cloud/services/rest/controllers/ProcessInstanceAdminControllerImplIT.java:
```diff
@@ -68,10 +73,10 @@
 public class ProcessInstanceAdminControllerImplIT {
 
-    private static final String DOCUMENTATION_IDENTIFIER = "process-instance";
+    private static final String DOCUMENTATION_IDENTIFIER = "process-admin";
```
@almerico can we call this process-instance-admin?

salaboy commented on activiti-cloud-starter-runtime-bundle/src/test/java/org/activiti/cloud/starter/tests/runtime/ProcessInstanceIT.java:
```diff
@@ -341,7 +341,7 @@ public void suspendShouldPutProcessInstanceInSuspendedState() {
         return responseEntity;
     }
-
+//almer
```
@almerico can we remove this one?

CTI777 committed:
Merge remote-tracking branch 'origin/develop' into almerico-2151-admin-resume-process

CTI777 committed:
fix: wrong indentation

CTI777 committed:
fix: documentation identifier renamed 

As requested here: https://github.com/Activiti/activiti-cloud-runtime-bundle-service/pull/145/files

CTI777 committed:
fest: test added for admin resume process

salaboy commented on activiti-cloud-starter-runtime-bundle/src/test/java/org/activiti/cloud/starter/tests/runtime/ProcessInstanceIT.java:
```diff
@@ -388,6 +388,45 @@ public void resumeShouldPutASuspendedProcessInstanceBackToActiveState() {
+        //testadmin should see process instances at admin endpoint
+        keycloakSecurityContextClientRequestInterceptor.setKeycloakTestUser("testadmin");
+        ResponseEntity<Void> responseEntity = adminExecuteRequestSuspendProcess(startProcessEntity);
+        
```
@CTI777 it wouldn't hurt to check that the process instance is SUSPENDED at this point.. just to make sure that the next check after resume (RUNNING) make sense

CTI777 replied:
Should be implemented

CTI777 committed:
feat: extra check that process is suspended
